app/views/invitations/new.html.haml

```diff
@@ -0,0 +1,21 @@
+.row
+  .col-md-12.pagecontainer
+    .pagetitle
+      %h3 Add subscriber to your website
```

Comment:
add subscriber to your website??? Add member/user to your website nu subscriber mai sergiule.... @Panioglo